ExampleUsage/EnumExample.cs

```diff
@@ -55,20 +59,98 @@ public void ExampleOfUsingApply()
 				context.Database.ExecuteSqlCommand(migrationSql);
 			}
 		}
-	}
+
+        [Test]        
+        public void CheckIfDescriptionColumnsExists()
```

Comment:
just noticed, this test is in the wrong project, this project is an example for users of the library of how it's intended to be used, not just generic unit test code, this needs to go in the EfEnumToLookupTests project